src/app/public/modules/timepicker/timepicker.interface.ts

```diff
@@ -1,9 +1,37 @@
 export interface SkyTimepickerTimeOutput {
+
+  /**
+   * Specifies the hour.
+   */
   hour: number;
+
+  /**
+   * Specifies the minute.
+   */
   minute: number;
+
+  /**
+   * Specifies the meridian.
+   */
   meridie: string;
+
+  /**
+   * Specifies the timezone.
+   */
   timezone: number;
+
+  /**
+   * Specifies the date in iso8601 format.
```

Comment:
Should `iso8601 format` be a link to something like https://www.iso.org/iso-8601-date-and-time-format.html?

Reply:
Yeah that works for me. Done.